server/controllers/admin/campaign/getDonorsByCampaignId.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, NextFunction } from 'express';
+import { Campaign, Donor } from '../../../database/models';
+import { CustomedError } from '../../../utils';
+import { paramsSchema } from '../../../utils/validation';
+
+const getDonorsByCampaignId = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const {
+      page = 1,
+    }: any = req.query;
+
+    const { id } = await paramsSchema.validateAsync(req.params);
+    const donors = await Campaign.findByPk(id, {
+      offset: (page - 1) * 6,
+      limit: 1,
+      include: {
+        model: Donor,
+        attributes: {
+          exclude: ['password', 'createdAt', 'updatedAt'],
+        },
```

Comment:
I think we here just the donors for campaigns so no need for campaign attributes so exclude